### PR TITLE
chore: add GitHub Action Test workflow

### PR DESCRIPTION
**Pull request by JackuB**
Replace Travis with GitHub Actions

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - run: npm install
+      - run: npm run lint
+      - run: npm test
```
Comment (ghost) [inline .github/workflows/release.yaml]: `npm test` already runs `npm run lint` so we're running it twice here.

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm run lint
+      - run: npm test
```
Comment (ghost) [inline .github/workflows/test.yaml]: Same thing. `npm test` already runs `npm run lint` so we're running it twice here.

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    strategy:
+      matrix:
+        node-version: [12, 14, 16]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
```
Comment (ghost) [inline .github/workflows/test.yaml:21]: Can add `cache: npm`. See: https://github.com/actions/setup-node#caching-packages-dependencies

**Review by ghost — APPROVED** [review]

File: .github/workflows/test.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+name: Test
+
+on: push
```
Comment (ghost) [inline .github/workflows/test.yaml]: Should we exclude master? We run tests again in the release workflow.

**Review by ghost — APPROVED** [review]

File: .github/workflows/test.yaml (hunk 3 above)
Comment (JackuB) [reply .github/workflows/test.yaml:21]: We can't cache without package lock

File: .github/workflows/test.yaml (hunk 4 above)
Comment (JackuB) [reply .github/workflows/test.yaml]: Updated

**Pull request merged**